SPMD_USER_GUIDE.md

```diff
@@ -0,0 +1,215 @@
+# User Guide: Running HuggingFace Llama 2 Training on v4 and v5e
+
+
+This user guide provides a concise overview of the essential steps required to run HuggingFace (HF) Llama 2 training on both v4 and v5e. It first demonstrates how to do so on barebone TPU VMs, and then shares a Dockerfile for those who would prefer to run it in a container.
+
+
+## Environment Setup
+
+The following setup assumes to run the training job with Llama 2 7B.
+
+
+### Cloud TPU VM Creation
+```
+export TPU_NAME=your-tpu-name
+export ZONE=us-central2-b
+export ACCELERATOR_TYPE=v4-32
+export RUNTIME_VERSION=tpu-ubuntu2204-base
+export PROJECT=your-project
+
+gcloud alpha compute tpus queued-resources create $TPU_NAME
+  \ --node-id=$TPU_NAME
```

Comment:
The `\` should be on the end of each line rather than beginning, right?

Reply:
Oops...